File: 프로그래머스/lv0/120902. 문자열 계산하기/문자열 계산하기.py
def solution(my_string):
    epr = my_string.split()
    answer = int(epr[0])
    for i in range(2, len(epr), 2):
        op = 1 if epr[i - 1] == "+" else -1 
        answer += op * int(epr[i])
    return answer

File: 프로그래머스/lv0/120902. 문자열 계산하기/문자열 계산하기.java
class Solution {
    public int solution(String my_string) {
        String[] epr = my_string.split(" ");
        int answer = Integer.parseInt(epr[0]);
        for (int i = 2; i < epr.length; i += 2) {
            int op = epr[i - 1].equals("+") ? 1 : -1;
            answer += op * Integer.parseInt(epr[i]);
        }
        return answer;
    }
}